solr/core/src/test/org/apache/solr/cloud/TestConfigSetsAPI.java

```diff
@@ -1121,10 +1121,6 @@ public void testDeleteErrors() throws Exception {
     DeleteNoErrorChecking delete = new DeleteNoErrorChecking();
     verifyException(solrClient, delete, NAME);
 
-    // ConfigSet doesn't exist
```

Comment:
Why remove?

Reply:
I removed the existence check from ConfigSetCmds.deleteConfigSet (as far as I understand, this was the suggestion here: https://issues.apache.org/jira/browse/SOLR-15421?focusedCommentId=17349401&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17349401 )

After that this test fails because it expects a failure. So I changed the test. Removed this part, and add  a similar test to the end of testDelete.

This means that this patch changes the behaviour of the delecteConfigSet commands. If this is bad, we can go back to the first version of this patch with two checkConfigExists methods.

Reply:
Ah, right.